package.json

```diff
@@ -57,75 +57,37 @@
     "@types/debug": "4.1.5",
     "@types/human-interval": "1.0.0",
     "@types/mongodb": "3.6.3",
-    "@typescript-eslint/eslint-plugin": "4.13.0",
-    "@typescript-eslint/parser": "4.13.0",
+    "@typescript-eslint/eslint-plugin": "^4.14.1",
+    "@typescript-eslint/parser": "^4.14.1",
     "blanket": "1.2.3",
     "coveralls": "3.1.0",
     "delay": "4.4.0",
     "eslint": "7.18.0",
-    "eslint-config-xo": "0.34.0",
-    "eslint-config-xo-typescript": "0.37.0",
+    "eslint-config-prettier": "^7.2.0",
     "eslint-plugin-ava": "11.0.0",
     "eslint-plugin-eslint-comments": "3.2.0",
     "eslint-plugin-import": "2.22.1",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-unicorn": "24.0.0",
```

Comment:
`eslint-plugin-unicorn` can be removed?

Reply:
Yes. You have to remove the eslint warning inside a file.

Reply:
If we are using eslint defaults, we can pretty much remove most of the things which start with "eslint-".
Am I right?

Reply:
Yes. But eslint default are strict on "any" types.
I changed the error to a warning.
We can fix that after we merge this